xdsl/interactive/app.py

```diff
@@ -157,7 +158,15 @@ class InputApp(App[None]):
     text areas.
     """
 
-    def __init__(self):
+    pre_loaded_file_path_option: str | None = None
```

Comment:
```suggestion
    input_text: str
```
In the init, this should probably be initialised to `INITIAL_IR_TEXT if self.input_text is None else self.input_text`